pgcontents/tests/test_synchronization.py

```diff
@@ -214,6 +214,13 @@ def encrypted_pgmanager(user_id):
         paths = [(user_id, path)
                  for user_id in user_ids
                  for path in populate(managers[user_id])]
+
+        # Create a text file for each user as well, which should be ignored by
+        # the notebook generators
+        model = {'content': 'text file contents', 'format': 'text'}
+        for manager in managers.values():
+            manager.new(model, path='text file.txt')
```

Comment:
The only way I know how to do it is with patching, so in `test_generate_files` for example you could do:
```python
import logging
    import mock
    with mock.patch('logging.Logger.warning') as mock:
        logger = logging.Logger('test')
        check_call({'logger': logger}, paths)
        mock.assert_called_with('Corrupted file with id 27 in table files.')
```